src/cljs/bundled/lumo/closure.cljs

```diff
@@ -2152,17 +2152,16 @@
 
 (defn build
   "Given a source which can be compiled, produce runnable JavaScript."
-  ([source opts]
+  ([source opts cb]
```

Comment:
this still needs the 2-arity function

Reply:
Why? Is it used? Do you prefer it? To avoid duplication, the default  callback is passed in.i can add it back if you want to

Reply:
JVM CLJS has a 2-arity version. I think we should have one too

Reply:
Ok np will add it back